docs/network-requirements.md

```diff
@@ -3,16 +3,17 @@
-| Antrea with VXLAN enabled     | All                 | UDP 4789                                   |                              |
-| Antrea with Geneve enabled    | All                 | UDP 6081                                   |                              |
-| Antrea with STT enabled       | All                 | TCP 7471                                   |                              |
-| Antrea with GRE enabled       | All                 | IP Protocol ID 47                          | No support for IPv6 clusters |
-| Antrea with IPsec ESP enabled | All                 | IP protocol ID 50 and 51, UDP 500 and 4500 |                              |
-| Antrea with WireGuard enabled | All                 | UDP 51820                                  |                              |
-| All                           | kube-apiserver host | TCP 443 or 6443\*                          |                              |
-| All                           | All                 | TCP 10349, 10350, 10351, UDP 10351         |                              |
+| Configuration                                          | Host(s)                    | ports/protocols                            | Other                        |
+|--------------------------------------------------------|----------------------------|--------------------------------------------|------------------------------|
+| Antrea with VXLAN enabled                              | All                        | UDP 4789                                   |                              |
+| Antrea with Geneve enabled                             | All                        | UDP 6081                                   |                              |
+| Antrea with STT enabled                                | All                        | TCP 7471                                   |                              |
+| Antrea with GRE enabled                                | All                        | IP Protocol ID 47                          | No support for IPv6 clusters |
+| Antrea with IPsec ESP enabled                          | All                        | IP protocol ID 50 and 51, UDP 500 and 4500 |                              |
+| Antrea with WireGuard enabled                          | All                        | UDP 51820                                  |                              |
+| Antrea with Multi-cluster with WireGuard encryption    | Multi-cluster Gateway Node | UDP 51821                                  |                              |
+| All                                                    | kube-apiserver host        | TCP 443 or 6443\*                          |                              |
+| All                                                    | All                        | TCP 10349, 10350, 10351, UDP 10351         |                              |
```

Comment:
I suppose you need a new PR to update MC docs for WireGuard related change?

Reply:
Sure, I will create a new PR later.